components/admin_console/group_settings/group_details/group_details.jsx

```diff
@@ -34,34 +34,39 @@ export default class GroupDetails extends React.PureComponent {
             link: PropTypes.func.isRequired,
             unlink: PropTypes.func.isRequired,
             patchGroupSyncable: PropTypes.func.isRequired,
+            patchGroup: PropTypes.func.isRequired,
         }).isRequired,
     };
 
     static defaultProps = {
         members: [],
         groupTeams: [],
         groupChannels: [],
-        group: {display_name: ''},
+        group: {display_name: '', allow_reference: false},
         memberCount: 0,
     };
 
     constructor(props) {
         super(props);
+        const group = props.group;
```

Comment:
I dont see the value in this creating this variable could just do 
`Boolean(props.group.allow_reference),` since there is only a single props attribute being referenced